### ISSUE TITLE
Submit to CRAN

### ISSUE THREAD

**Issue by mdlavallee92** [OHDSI/Capr#50]
do cran submission using `devtools::submit_cran()`. before that run `devtools::check_win_release()`

**Comment by ablack3** [comment]
Any update here?

**Comment by ablack3** [comment]
PR here: https://github.com/OdyOSG/Capr/pull/9

**Comment by ablack3** [comment]
update?

**Comment by mdlavallee92** [comment]
Agreed, it would be nice to get this package on CRAN but at this point in time I don't see it as a priority or within my bandwidth as a developer. This package is already available publicly and via HADES. If you would like to help with the preparation to CRAN please feel free. 

Issues I see as more of a priority are:
- improving the documentation and explanation of how to use the package correctly
- improving the object prints on the console
- adding validity checks to the objects
- reintroducing the capr call function and capr save
- fixing bugs in censor events such as scenarios with Any death or any visit

**Comment by mdlavallee92** [comment]
Reopenning this issue now that CirceR is on CRAN

**Comment by rbcavanaugh** [comment]
Hey @mdlavallee92 Are there any areas you need help with -particularly related to tutorials/documentation? I'd really love to work capR into our workflow and avoid ATLAS altogether but I'm having trouble making the jump with the sparse documentation and my admittedly non-comprehensive knowledge of ATLAS.

**Comment by mdlavallee92** [comment]
I would appreciate help. Documentation always falls to the wayside 😅. I will reach out to you individually to see where we can collaborate
